doc/usermanual.rst

```diff
@@ -639,25 +664,14 @@ density profile of the collision partner(s).
       density[n] = f(x,y,z);
     }
 
-LIME can deal with an unlimited number of collision partners (n).
-However, LIME will produce an error if more density profiles are given
-in the density subroutine than there are collision partners listed in
-the molecular data file. In most cases, a single density profile will
-suffice. The density is a number density, that is, the number of
-collision partners per unit volume (in cubic meters, not cubic
-centimeters). Please note that the current version of LIME always takes
-the abundance relative to the first collision partner. This is
-potentially a problem if the first collision partner is not the total H2
-density and the user will have to correct for this in the abundances
-(see below).
+LIME can deal with an unlimited number n of collision partners (CPs). In most cases, a single density profile will suffice. Note that the number of returned density function values no longer has to be the same as the number of CPs listed in the moldata file(s) so long as the user sets values for the collPartIds parameter, but if this parameter is not supplied, and the numbers are different, LIME may not be able to match the CPs associated with each density value to those in the moldata file(s). Note also that moldata CPs for which there is no matching density will be ignored.
```

Comment:
> unlimited number n of collision partners

Isn't this limited by  `MAX_N_COLL_PART`?